### PR TITLE
[rust] Update Selenium Manager roadmap in README

### PR DESCRIPTION
**Pull request by bonigarcia**
### Description
This PR updated the roadmap (i.e., milestone planed) for the initial development of Selenium Manager.

### Motivation and Context
Part of the Selenium Manager development.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [X] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: rust/README.md [hunk 1]
```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
...
+| M1: Driver management                   | <ul><li>Beta version of the Selenium Manager.</li> <li>Focused on driver management for Chrome, Firefox, and Edge.</li> <li>Selenium Manager compiled for Windows, Linux, and macOS (in GH Actions).</li> <li>Available in Selenium binding languages (Java, JavaScript, Python, Ruby, and C#).</li> <li>Used as a fallback for language bindings, when the driver is not found.</li> <li>Selenium Manager binaries bundled within the binding languages.</li></ul> |
+| M2: Driver management for IEDriver      | <ul><li>Include driver support for IExplorer.</li> <li>Consider msedgedriver in IE Mode.</li></ul>                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (titusfortner) [inline rust/README.md]: msdgedriver in IE Mode uses the IEDriver classes and driver, so there's nothing that Manager needs to do.

File: rust/README.md [hunk 2]
```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
...
+|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| M1: Driver management                   | <ul><li>Beta version of the Selenium Manager.</li> <li>Focused on driver management for Chrome, Firefox, and Edge.</li> <li>Selenium Manager compiled for Windows, Linux, and macOS (in GH Actions).</li> <li>Available in Selenium binding languages (Java, JavaScript, Python, Ruby, and C#).</li> <li>Used as a fallback for language bindings, when the driver is not found.</li> <li>Selenium Manager binaries bundled within the binding languages.</li></ul> |
```
Comment (titusfortner) [inline rust/README.md:75]: msedgedriver isn't working right now; I've set it to ignore in the bindings, I say we push it off to M2, but depends on how hard is the fix.

File: rust/README.md [hunk 3]
```diff
@@ -70,15 +70,11 @@ Alternatively, you can replace `cargo run` with `bazel run //rust:selenium-manag
...
+| M2: Driver management for IEDriver      | <ul><li>Include driver support for IExplorer.</li> <li>Consider msedgedriver in IE Mode.</li></ul>                                                                                                                                                                                                                                                                                                                                                                  |
+| M3: Rich configuration                  | <ul><li>Proxy support in Selenium Manager.</li> <li>Extra configuration capabilities from binding languages to Selenium Manager (e.g., force to use a given driver version, etc.).</li></ul>                                                                                                                                                                                                                                                                        |
```
Comment (titusfortner) [inline rust/README.md:77]: Add ability to specify browser location (Se Mgr will check that location for version and grab the right driver for it).
Do we support beta/dev/canary right now?

**Review by titusfortner — CHANGES_REQUESTED** [review]
Also need to be able to support M1 architecture; cross-compilation?
How are we determining 32 vs 64 right now? Should that be configurable?
Eventually need to figure out if we can solve the snap issue for people.

File: rust/README.md (hunk 1 above)
Comment (bonigarcia) [reply rust/README.md]: Updated

File: rust/README.md (hunk 2 above)
Comment (bonigarcia) [reply rust/README.md:75]: This is the first notice I have about that. What is the problem? In any case, I think this PR is not the best place to report a issue like that. If I know the problem, I can try to solve it for M1.

File: rust/README.md (hunk 3 above)
Comment (bonigarcia) [reply rust/README.md:77]: Not now, but it should be easy to implement. That can be part of the extra configuration milestone.

**Comment by bonigarcia** [comment]
Regarding ARM (or other architectures), I think that is not priority. I tried the Selenium Manager macOS binary (generated in GH Actions) in a macOS M1 and it works. Regarding 32-64 arch, I am not sure. It would be interesting to test the current binaries (Windows, macOS, Linux) generated in GH Actions in different platforms.

**Comment by titusfortner** [comment]
> I think that is not priority

I'm not saying it is a priority, but it should be on a list of possible features. Maybe not part of "roadmap" per se.

**Review by titusfortner — APPROVED** [review]

**Pull request merged**